FHEM/00_SIGNALduino.pm

```diff
@@ -1358,6 +1358,21 @@ my %ProtocolListSIGNALduino  = (
 			length_min      => '48',
 			length_max      => '48',
 		},
+	"72" => ## FHT80B  (Funk-Heizkörperthermostat)
+		{
+			name         	=> 'FHT80',
+			comment		=> 'FHT80',
```

Comment:
mit einem 
`get sduino protocolIDs`
kannst Du erkennen, daß es keinen Sinn macht in "comment" das selbe reinzuschreiben wie in "name"
Bei comment kann man z.B. bei den 868 MHz Sensoren '868 MHz' reinschreiben.